test/test_store_manager_caches.py

```
@@ -0,0 +1,481 @@
+        # The goal of this test is to prove the tombstone does NOT gate
+        # explicit install_plugin calls. The full install path has a bunch
+        # of downstream validation (manifest fields, dependency install,
+        # etc.) that we don't need to exercise here — the "tombstone did
+        # not block" invariant is proven purely by the fact that
+        # install_plugin reaches the download step at all. If the tombstone
+        # had been gating, install_plugin would have returned False before
+        # calling _install_via_git.
+        try:
+            self.sm.install_plugin("bar")
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)